cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -737,55 +737,106 @@
   <!-- Kafka Server Configuration -->
 
   <property>
-    <name>kafka.bind.address</name>
-    <value>0.0.0.0</value>
+    <name>kafka.bind.port</name>
```

Comment:
shouldn't this be `kafka.server.port` (and deprecate the old `kafka.bind.port`)?